tests/unit/test_zigzag_test_log.py

```diff
@@ -302,6 +302,47 @@ def test_successful_test_case_attachments(self, single_passing_xml, mocker):
         assert len(tl.qtest_test_log.attachments) == 1
         assert tl.qtest_test_log.attachments[0].content_type == 'application/xml'
 
+    def test_passing_test_empty_failure_link(self, single_passing_xml, mocker):
+        """Test to ensure that the failure_link is empty when a test case passes.
+        """
+        qtest_id = 123456789
+        search_response = {
+            "links": [],
+            "page": 1,
+            "page_size": 100,
+            "total": 1,
+            "items": [
+                {
+                    "id": qtest_id,
+                    "name": "PRO-18405 Fake!"
+                }
+            ]
+        }
+
+        # a mock for a ZigZag object
+        zz = mocker.MagicMock()
+        zz._qtest_api_token = 'totally real'
+        zz._qtest_project_id = '54321'
+
+        # patch the API calls
+        mock_post_response = mocker.Mock(spec=requests.Response)
+        mock_post_response.text = json.dumps(search_response)
+        mocker.patch('requests.post', return_value=mock_post_response)
+        mock_field_resp = mocker.Mock(spec=swagger_client.FieldResource)
+        mock_field_resp.id = 12345  # this will be the id of the failure output field
+        mock_field_resp.label = 'Failure Output'
```

Comment:
It's not clear what's being tested here because the test indicates that the subject is 'Failure Link', but all I see are references to 'Failure Output'.

Reply:
I think this is just a bad test now.  I'll remove.